SN-Copilot integration

```diff
@@ -0,0 +1 @@
+As a product manager preparing for Microsoft Copilot integration with ServiceNow, I need to set up Agile Scrum for my team. The document should cover roles, product vision, themes, epics, backlog prioritization, and sprint planning. It must ensure clear priorities and readiness for a seamless integration process.
```

Comment:
Please consider aligning this prompt to the RISEN framework. I've checked off the components that are currently reflected in this prompt:

- [ ] R — Role: Define the AI's role. (e.g., "Act as an agile coach.")
- [x] I — Instructions: Provide clear instructions. (e.g., "Write a guide on crafting a good agile development story.")
- [x] S — Steps: Outline specific steps. (e.g., "Start with a user persona, describe the user's goal, list the tasks to achieve the goal, and define the acceptance criteria.")
  - Steps are not very clear
- [ ] E — End goal: State the goal and audience. (e.g., "Help team members understand how to write clear and effective agile stories.")
- [ ] N — Narrowing: Add constraints. (e.g., "Max 800 words, avoid technical jargon, use practical examples.")